### DISCUSSION TITLE
Deploy Error: `Resource is limited - try again in 20 hours (more than 100, code: "api-deployments-free-per-day").`

### DISCUSSION THREAD

**Discussion by TylerAPfledderer** [vercel/vercel#10748] [Help]
### Question

Hello!

When running vercel preview check for any PR under a personal project (Hobby account), I am receiving the error
```
Resource is limited - try again in 20 hours (more than 100, code: "api-deployments-free-per-day").
```

I find this odd, because I don't run anywhere close to enough deploys to hit the maximum for a hobby account.

Please see [this repo affected](https://github.com/TylerAPfledderer/tylerpweb.dev). Thank you.

**Comment by Abdel-Monaam-Aouini** [comment]
Is there any solution here?

**Reply by SilianZ** [reply]
See this document: https://vercel.com/docs/limits/overview

**Reply by jkm96** [reply]
Hello, I am facing the same error on Hobby plan yet I have only had < 50 deployments within the last 15 days. The docs says I should have a total of 100 deployments per day. Kindly advise.

**Comment by rkat7** [comment]
I'm facing the same issue :/
